src/main/javatests/com/google/summit/serialization/SerializationTest.kt

```diff
@@ -34,7 +34,7 @@ class SerializationTest {
   @Test
   fun testSerialization_compilationUnit() {
     val ser = Serializer(format = true)
-    val expectedJson = readTestFile("mixednodes.json")
+    val expectedJson = readTestFile("mixednodes.json").trimEnd()
```

Comment:
would it make sense to change readTestFile to trim the end (or, if not, to have a method that does it)?